CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age,
+body size, disability, ethnicity, gender identity and expression, level of
+experience, nationality, personal appearance, race, religion, or sexual
+identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+* advances
```

Comment:
Your bullets are broken here. (You can download the official markdown from https://www.contributor-covenant.org/version/1/4/code-of-conduct.md)

Reply:
This file was automatically generated by GitHub and [renders](https://github.com/drewwalters96/crust/blob/fc9162a9d1e31e5fadc320551102e5156ac3ceed/CODE_OF_CONDUCT.md).

I can change the bullet syntax if you insist on this change being made.

Reply:
It does not render correctly. See how "address, without explicit permission" and "professional setting
" and "advances" are separate bullets?

Reply:
My mistake; you are correct. I will amend according to the link you sent.